stores/post_store.jsx

```diff
@@ -202,6 +203,7 @@ PostStore.dispatchToken = AppDispatcher.register((payload) => {
         dispatch({...action, type: ActionTypes.EDIT_POST});
         break;
     case ActionTypes.RECEIVED_POST_SELECTED:
+        dispatch({data: action.postId, type: PostTypes.RECEIVED_POST_SELECTED});
```

Comment:
I'm not super familiar with our Flux->Redux compatibility layer. Can you elaborate on this addition?

Reply:
Yes. `ActionTypes.RECIEVED_POST_SELECTED` is managed by the AppDispatcher, and is an "Old-flux" event which is not managed by redux at all. In the redux we have a reducer to manage `PostTypes.RECEIVED_POST_SELECTED` to update the `selectedPost` in the global store, that information was used by the `getUsersTyping` selector, but in not generated by the webapp (before this change). Finally I'm using my own selector which don't use the selectedPost information in the webapp, but anyway, makes sense to dispatch that event to maintain the consistency. I think.